pkg/crc/machine/machine.go

```diff
@@ -53,14 +54,26 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		Memory:     startConfig.Memory,
 	}
 
-	logging.InfoF("Extracting the Bundle tarball ...")
+	logging.InfoF("Extracting the %s Bundle tarball ...", filepath.Base(machineConfig.BundlePath))
 	crcBundleMetadata, extractedPath, err := bundle.GetCrcBundleInfo(machineConfig)
 	if err != nil {
 		logging.ErrorF("Error to get bundle Metadata %v", err)
 		result.Error = err.Error()
 		return *result, err
 	}
 
+	// Verify, user is using latest bundle with released version of crc.
+	if constants.DefaultHostname != crcBundleMetadata.Nodes[0].Hostname {
```

Comment:
this is not a valid check, as it assume a single situation.

Reply:
feels more like a semantic version scheme is needed...

Reply:
OK, as I understand... you check the cached/running instance node hostname to the one from the bundle... right? If so, make this more obvious by extracting this in a named function,